pkgs/top-level/all-packages.nix

```diff
@@ -16325,6 +16325,8 @@ with pkgs;
 
   spooles = callPackage ../development/libraries/science/math/spooles {};
 
+  spr= callPackage ../development/tools/spr {};
```

Comment:
```suggestion
  spr = callPackage ../development/tools/spr {};
```

Reply:
Thanks @fabaff! I fixed this.

(Also fixed a Darwin build error below, so all looks okay now.)